drydock_backups/docker-entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+set -eo pipefail
+
+FILENAME="$(date +'%Y-%m-%d').gz"
+
+if [ "$1" = 'mysql' ]; then
+      echo "1"
+      mysqldump -u $MYSQL_ROOT_USERNAME -h $MYSQL_HOST -P $MYSQL_PORT --password=$MYSQL_ROOT_PASSWORD \
+            --all-databases --single-transaction --flush-logs | gzip > $FILENAME
+elif [ "$1" = 'mongo' ]; then
+      if [[ -z "${MONGODB_USERNAME}" ]]; then
+            mongodump \
+            --host $MONGODB_HOST:$MONGODB_PORT \
+            -d $MONGODB_DATABASES --gzip \
+            --archive=$FILENAME
+      else
+            mongodump --username $MONGODB_USERNAME --password $MONGODB_PASSWORD --authenticationDatabase=admin \
```

Comment:
I think we can build a [uri](https://www.mongodb.com/docs/manual/reference/connection-string/) instead?

we can add something like

```bash
MONGODB_URI="mongodb://"
MONGODB_PORT=${MONGODB_PORT:-27017}
MONGODB_HOST=${MONGODB_HOST:-mongodb}
MONGODB_AUTHDB=${MONGODB_AUTHDB:-admin}

if [ -n "${MONGODB_USERNAME}" ] && [ -n "${MONGODB_PASSWORD}" ]; then
    MONGODB_URI+="${MONGODB_USERNAME}:${MONGODB_PASSWORD}@"
fi

MONGODB_URI+="${MONGODB_HOST}:${MONGODB_PORT}/${MONGODB_DATABASES}?authSource=${MONGODB_AUTHDB}"
```

at the start of the file and do something like:

```bash
elif [ "$1" = 'mongo' ]; then
      
    mongodump --uri="${MONGODB_URI}" --gzip --archive=$FILENAME
else
```

I think we shouldn't set the `MONGODB_DATABASES` value by default, at the moment we only are dumping the DB used by edxapp and if forum uses a different DB then it's not included in the backup. If we leave the database blank (e.g. `mongodb://root:secure-password@mongodb:27017/?authSource=admin`) it should dump all the DBs.